### PR TITLE
CRN-969 | Backend work to enable users to dismiss "Getting Started" dialog.

### PR DESCRIPTION
**Pull request by m-doughty**
# CRN-969 | Backend work to enable users to dismiss "Getting Started" dialog

## JIRA Ticket

[CRN-969](https://asaphub.atlassian.net/jira/software/c/projects/CRN/boards/8?modal=detail&selectedIssue=CRN-969)

## Description of Changes

- Adds the ability to set and retrieve `dismissedGettingStarted` toggle in Squidex
- Updates the Squidex schema to add the boolean property
- Removes the feature flag
- Updates the subtext

## Testing Instructions

These tests can be performed on the [PR Branch](https://1986.hub.asap.science/).

1. Log in
2. On the home page, you will see the 'Getting Started with ASAP' section
3. Click the button labelled 'Don't Show Again' and confirm on the modal
4. You should see the section disappear, and the modal should dismiss itself
5. The subtext of the header should also be removed

If you need to re-perform the test, you will need to log into Squidex and toggle `dismissedGettingStarted` off for your

### CODE REVIEW

**Review by kmaid — CHANGES_REQUESTED** [review]

File: packages/react-components/src/templates/__tests__/DashboardPageHeader.test.tsx [hunk 1]
```diff
@@ -22,3 +22,27 @@ it('displays user first name in welcome mesage', () => {
     `"Welcome to the Hub, John!"`,
   );
 });
+
+describe('switches off the subtext if user dismissed the getting started text', () => {
+  it('Correctly shows the subtext if dismissedGettingStarted is false', () => {
+    render(
+      <DashboardPageHeader
+        firstName={'Mike'}
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/DashboardPageHeader.test.tsx:30]:
```suggestion
        firstName="Mike"
```

File: apps/crn-server/src/data-providers/users.data-provider.ts [hunk 2]
```diff
@@ -492,6 +502,11 @@ export const parseGraphQLUserToDataObject = (
         : undefined,
     createdDate,
     orcid,
+    dismissedGettingStarted:
```
Comment (peterstarling) [inline apps/crn-server/src/data-providers/users.data-provider.ts:505]: wouldnt this be enough?
```suggestion
    dismissedGettingStarted: !!item.flatData?.dismissedGettingStarted
```

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: apps/crn-server/src/data-providers/users.data-provider.ts (hunk 2 above)
Comment (m-doughty) [reply apps/crn-server/src/data-providers/users.data-provider.ts:505]: Committed.

File: apps/crn-frontend/src/dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -41,17 +41,22 @@ const Dashboard: FC<Record<string, never>> = () => {
   usePrefetchCalendars();
   const patchUser = usePatchUserById(id);
   const user = useUserById(id);
+  const dismissedGettingStarted =
+    user && typeof user.dismissedGettingStarted === 'boolean'
+      ? user.dismissedGettingStarted
+      : false;
```
Comment (kmaid) [inline apps/crn-frontend/src/dashboard/Dashboard.tsx:47]:
```suggestion
```

File: packages/react-components/src/templates/DashboardPageHeader.tsx [hunk 4]
```diff
@@ -20,23 +20,27 @@ const textStyles = css({
 
 type DashboardPageHeaderProps = {
   readonly firstName?: string;
+  readonly dismissedGettingStarted?: boolean;
 };
 
 const DashboardPageHeader: React.FC<DashboardPageHeaderProps> = ({
   firstName,
+  dismissedGettingStarted,
```
Comment (kmaid) [inline packages/react-components/src/templates/DashboardPageHeader.tsx:28]:
```suggestion
  dismissedGettingStarted = false,
```

**Pull request merged**